.gitignore

```diff
@@ -51,6 +51,7 @@ packages/**/lib
 generated
 storybook-static
 tsconfig.tsbuildinfo
+lib/
```

Comment:
apps/cli/lib

Reply:
why adding here something that is specific to one app? it can be a local .gitignore file unless its literally needed everywhere